models/account.js

```diff
@@ -1,7 +1,7 @@
 
 module.exports = (sequelize, DataTypes) => {
   const Account = sequelize.define('Account', {
-    membershipId: DataTypes.INTEGER,
+    membershipNoId: DataTypes.INTEGER,
```

Comment:
@albert85 should this be `membershipNoId` or `membershipNo`?  Making it the former would mean that there is a table holding memebership ids